docs/npm.md

```diff
@@ -132,3 +133,15 @@ A step-by-step example of how you might do this is:
 2. Navigate into the directory: `cd /usr/local/Cellar/code-server/<version>/libexec/lib/vscode/`
 3. Recompile the native modules: `npm rebuild`
 4. Restart code-server
+
+### Debugging install issues with npm
+
+Unfortunately, `yarn` swallows all logs on `yarn global add` so instead you can install with `npm` and see the logs:
```

Comment:
```suggestion
`yarn` suppresses logs when running `yarn global add`, so to debug installation issues, install with `npm` instead:
```

Reply:
oh noo! I didn't get a chance to add this because I hit "enable auto-merge" - I'll follow up with a PR